packages/dropdownMenu/components/DropdownMenuItem.tsx

```diff
@@ -9,12 +10,49 @@ export interface DropdownMenuItemProps {
    * The value that the item represents
    */
   value: string;
+  /**
+   * Whether to allow other actions on the item
+   */
   disabled?: boolean;
+  /**
+   * The length of all DropdownMenuItems in its parent DropdownMenuSection
+   */
+  listLength?: number;
+  /**
+   * The index within the DropdownMenuSection
+   */
+  index?: number;
+  /**
+   * If the current item is in use
+   */
+  isActive?: boolean;
+  /**
+   * Children for PopoverListItem
+   */
   children: React.ReactNode;
 }
 
-const DropdownMenuItem = ({ children }: DropdownMenuItemProps) => (
-  <>{children}</>
-);
+const DropdownMenuItem = ({
+  appearance,
+  disabled,
+  listLength,
+  isActive,
+  index,
+  children,
+  ...rest
+}: DropdownMenuItemProps) => {
+  return (
+    <PopoverListItem
+      listLength={listLength || 0}
+      isActive={isActive}
+      appearance={appearance}
+      index={index || 0}
+      disabled={disabled}
+      {...rest}
```

Comment:
💡 If we would set default arguments for `listLength` and `index` we could do:
`<PopoverListItem ...rest listlength={listlength} index={index}  />`

We would not need to spread them, except for the children as we do not do anything with these props.

Reply:
But not in this component as far as I can see?

This component could look like this:


```
const DropdownMenuItem = ({
  listLength = 0,
  index = 0,
  children,
  ...rest
}: DropdownMenuItemProps) => {
  return (
    <PopoverListItem
      listLength={listLength}
      index={index}
      {...rest}
    >
      <>{children}</>
    </PopoverListItem>
  );
};
```